### PR TITLE
[VSCode Extension] Adding stablecode-3b-int8 model

### PR DESCRIPTION
**Pull request by kumarijy**
made changes to the relevant files in order to add stablecode-3m model to OV code completion extension @apaniukov

### CODE REVIEW

File: modules/openvino_code/package.json [hunk 1]
```diff
@@ -171,7 +171,8 @@
             "default": "codet5p-220m-py",
             "enum": [
               "codet5p-220m-py",
-              "decicoder-1b-openvino-int8"
+              "decicoder-1b-openvino-int8",
+              "stablecode-completion-3b"
```
Comment (apaniukov) [inline modules/openvino_code/package.json:175]: Please add int8 to the model name: `"stablecode-completion-3b-int8"`

File: modules/openvino_code/shared/model.ts [hunk 2]
```diff
@@ -3,19 +3,24 @@ import { Features } from './features';
...
 export const MODEL_NAME_TO_ID_MAP: Record<ModelName, ModelId> = {
   [ModelName.CODE_T5_220M]: ModelId.CODE_T5_220M,
   [ModelName.DECICODER_1B_OPENVINO_INT8]: ModelId.DECICODER_1B_OPENVINO_INT8,
+  [ModelName.STABLECODE_COMPLETION_ALPHA_3B_4K_OPENVINO_INT8]: ModelId.STABLECODE_COMPLETION_ALPHA_3B_4K_OPENVINO_INT8,
 };
 
 export const MODEL_SUPPORTED_FEATURES: Record<ModelName, Features[]> = {
   [ModelName.CODE_T5_220M]: [Features.CODE_COMPLETION],
   [ModelName.DECICODER_1B_OPENVINO_INT8]: [Features.CODE_COMPLETION, Features.SUMMARIZATION],
+  [ModelName.STABLECODE_COMPLETION_ALPHA_3B_4K_OPENVINO_INT8]: [Features.CODE_COMPLETION],
```
Comment (apaniukov) [inline modules/openvino_code/shared/model.ts:25]: The model might support the summarization feature too, you need to check it.

File: modules/openvino_code/package.json
Comment (yatarkan) [inline modules/openvino_code/package.json:1]: Please increment extension patch version in `package.json` file (L4, `"version"` field) from `0.0.5` to `0.0.6`

File: modules/openvino_code/shared/model.ts [hunk 3]
```diff
@@ -3,19 +3,24 @@ import { Features } from './features';
 enum ModelId {
   CODE_T5_220M = 'Salesforce/codet5p-220m-py',
   DECICODER_1B_OPENVINO_INT8 = 'chgk13/decicoder-1b-openvino-int8',
+  STABLECODE_COMPLETION_ALPHA_3B_4K_OPENVINO_INT8 = 'chgk13/stablecode-completion-alpha-3b-4k-openvino-int8',
 }
 
 export enum ModelName {
   CODE_T5_220M = 'codet5p-220m-py',
   DECICODER_1B_OPENVINO_INT8 = 'decicoder-1b-openvino-int8',
+  STABLECODE_COMPLETION_ALPHA_3B_4K_OPENVINO_INT8 = 'stablecode-completion-3b',
```
Comment (apaniukov) [inline modules/openvino_code/shared/model.ts:12]: Please add `-int8` here too.

Comment (yatarkan) [inline modules/openvino_code/shared/model.ts:12]: New model name should also be added to side panel select options: `side-panel-ui/src/components/sections/ServerSection/ModelSelect/ModelSelect.tsx` (`const options: SelectOptionProps<ModelName>[]`, L6-L9)

Comment (yatarkan) [reply modules/openvino_code/shared/model.ts:12]: Also, I would suggest to make it more dynamic, e.g.
```typescript
const options: SelectOptionProps<ModelName>[] = Object.values(ModelName).map((value) => ({ value }));
```

**Comment by apaniukov** [comment]
It seems like the old `package-lock.json` has been deleted. @kumarijy include the new one in the PR.

**Review by apaniukov — APPROVED** [review]

**Pull request merged**